admin/serve-doc

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/python
 from __future__ import print_function
 
-import SimpleHTTPServer
-import SocketServer
+try:
+    # Python 3
+    import http.server as server
+    import socketserver
+except ImportError:
+    # Python2
```

Comment:
can we just drop python2 support?

Reply:
@tchaikov I wasn't brave enough to make that call. Seems every time I do it comes back to bite me so I went out of my way to make it backwards compatible. Are we definitely py3 only and completely on master now?

Reply:
will do